File: java/ql/src/Metrics/Callables/StatementNestingDepth.java
public static void printCharacterCodes_Bad(String[] strings) {
    if (strings != null) {
        for (String s : strings) {
            if (s != null) {
                for (int i = 0; i < s.length(); i++) {
                    System.out.println(s.charAt(i) + "=" + (int) s.charAt(i));
                }
            }
        }
    }
}

File: csharp/ql/src/Metrics/Callables/StatementNestingDepth.cs
public void printCharacterCodes(string[] strings)
{
    if (strings != null)
    {
        foreach(string s in strings){
            if (s != null)
            {
                foreach (char c in s)
                {
                    Console.WriteLine(c + "=" + (int)c);
                }
            }
        }
    }
}
